code/GenerateAnnotatedLibrary.java

```diff
@@ -20,7 +20,7 @@
 		SSR.txt (list of sequences identified as SSRs, and the SSR they matched)
 		protein.txt (list of the sequences identified as proteins and the protein they matched)
 		//LA4v2-satellite.fa (a satellite sequence for which no consensus sequence was found)
-		/home/a1635743/RepBase20.04.fasta/*rep.ref (RepBase libraries to base 
+		/home/a1635743/RepBase_lib/*.rep.ref (RepBase libraries to base 
```

Comment:
What's the basis for this change?

Reply:
@kortschak James thought this is a fasta file instead of a directory that contains RepBase library.